recOrder/plugin/workers/acquisition_workers.py

```diff
@@ -906,6 +901,54 @@ def work(self):
         self.phase_image_emitter.emit(phase)
         self.meta_emitter.emit(meta)
 
+    def _check_exposure(self) -> None:
```

Comment:
I like pulling this out as it's own function...looks good and makes sense. Nice docs, too.